examples/java/src/main/java/org/apache/beam/examples/WordCount.java

```diff
@@ -196,6 +197,26 @@ public String create(PipelineOptions options) {
       }
     }
 
+    /**
+     * Return default input file path according to runner type.
+     *
+     * <p><ul>
+     *   <li>SparkRunner:
+     *   .../src/test/resources/LICENSE</li>
+     *   <li>other runners:
+     *   gs://dataflow-samples/apache/LICENSE</li>
+     * </ul>
+     */
+    public static class InputFactory implements DefaultValueFactory<String> {
+      @Override
+      public String create(PipelineOptions options) {
+        if (options.getRunner().isAssignableFrom(SparkRunner.class)) {
+          return WordCount.class.getClass().getResource("/LICENSE").getPath();
```

Comment:
Use Resources.getResource to be context classloader aware.
https://google.github.io/guava/releases/snapshot/api/docs/com/google/common/io/Resources.html#getResource(java.lang.Class,%20java.lang.String)